lib/world.rb

```diff
@@ -37,8 +52,28 @@ def count_live_neighbours(position)
     @cells.count {|cell| all_neighbouring_positions.include?(cell.position)}
   end
 
-  def ideal_number_of_neighbours(count)
+  def positions_of_potential_resuscitating_cells
+    all_positions_around_cells = []
+    @cells.each { |cell|
+      neighbouring_positions(cell.position).each { |position| all_positions_around_cells.push(position)}
+    }
+    all_positions_around_cells.select {|empty_position| !cells_positions.include?(empty_position)}.uniq
+  end
+
+  def cells_positions
+    @cells.map {|cell| cell.position }
+  end
+
+  def number_of_neighbours_to_survive(count)
     count == 2 || count == 3
   end
 
+  def number_of_neighbours_to_resuscitate(count)
```

Comment:
I think the name isn't quite right. It sounds like it returns the number of neighbours necessary to resuscitate. But instead it checks if the neighbour count is correct to resuscitate the cell. Maybe change the name to `enough_neighbours_to_resuscitate?`